superset/config.py

```diff
@@ -743,7 +743,7 @@ def _try_json_readsha(filepath: str, length: int) -> Optional[str]:
 
 class CeleryConfig:  # pylint: disable=too-few-public-methods
     broker_url = "sqla+sqlite:///celerydb.sqlite"
-    imports = ("superset.sql_lab", "superset.tasks")
+    imports = "superset.sql_lab"
```

Comment:
```suggestion
    imports = ("superset.sql_lab")
```

Reply:
precommit does not like the parenthesis  if it's just one.

Reply:
can you do `("superset.sql_lab",)`?

Reply:
yep that worked.